pkg/bootstrap/os/module.go

```diff
@@ -99,6 +100,7 @@ func (c *ClearOSEnvironmentModule) Init() {
 		Name:     "StopETCDService",
 		Desc:     "Stop etcd service",
 		Hosts:    c.Runtime.GetHostsByRole(common.ETCD),
```

Comment:
@24sama 
I found that if the etcd group is empty, it will be panic when initializing the task. Can we optimize it?